NEWS

```diff
@@ -3,6 +3,16 @@ testtools NEWS
 
 Changes and improvements to testtools_, grouped by release.
 
+Next
+~~~~
+
+Changes
+-------
+
+* Python 2.6 and 3.2 support now gone. If you want to use either of these versions of
```

Comment:
A minor nit-pick, but this sentence isn't as wonderful as it could be, what about:

Python 2.6 and 3.2 are no longer supported. If you want....